src/tiles/grid/mediatype-icons.ts

```diff
@@ -0,0 +1,71 @@
+import { css, CSSResultGroup, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { audioIcon } from '../../assets/img/icons/mediatype/audio';
+import { etreeIcon } from '../../assets/img/icons/mediatype/etree';
+import { imagesIcon } from '../../assets/img/icons/mediatype/images';
+import { filmIcon } from '../../assets/img/icons/mediatype/film';
+import { softwareIcon } from '../../assets/img/icons/mediatype/software';
+import { textsIcon } from '../../assets/img/icons/mediatype/texts';
+import { tvIcon } from '../../assets/img/icons/mediatype/tv';
+import { videoIcon } from '../../assets/img/icons/mediatype/video';
+import { webIcon } from '../../assets/img/icons/mediatype/web';
+
+@customElement('mediatype-icon')
+export class MediatypeIcon extends LitElement {
```

Comment:
Q: Is this a duplicate of the `MediatypeIcon` above?

Reply:
yes, will refactor this when we merge the other component